packages/babel-cli/package.json

```diff
@@ -24,12 +24,12 @@
     "fs-readdir-recursive": "^1.1.0",
     "glob": "^7.0.0",
     "lodash": "^4.17.13",
-    "make-dir": "^2.1.0",
-    "slash": "^2.0.0",
+    "make-dir": "^3.0.0",
```

Comment:
Bumping the requirement to `>10.12.0` would allow babel to drop `make-dir` in favor of using `fs.mkdirSync(dir, {recursive: true})` directly.  10.13.0 was the first LTS, I would suggest babel should just specify that is the minimum version since dependencies which require node.js `>=10` are likely to only support LTS versions.